modules/tutorials/pages/pattern-matching/load-data.adoc

```diff
@@ -6,7 +6,7 @@ Below, we use the GSQL loading language to define a loading job script, which en
 
 The script to load the LDBC-SNB data is below.
 
-.# GSQL script for loading LDBC-SNB data
+.GSQL script for loading LDBC-SNB data
```

Comment:
The reason why this line began with # is because it is supposed to be a comment line in the code.  # was my favorite comment marker, but sadly it is deprecated.  Either
1. Put the line into the code block where it belongs, beginning with //
or 
2. Remove the line as unncessary.